core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java

```diff
@@ -2659,19 +2758,37 @@ public void createTable(final TransactionState transactionState, HTableDescripto
             else {
               admin.createTable(desc);
             }
+            // make sure the table is enabled
+            boolean keepPolling = true;
+            TableName tableName = desc.getTableName();
+            while (keepPolling) {
+
+               try {
+                 Thread.sleep(1000); // sleep one second or until interrupted
+               }
+               catch (InterruptedException e) {
+                  // ignore the interruption and keep going
+               }
+
```

Comment:
Generally table is enabled as part of admin coming back from a waited call. If not enabled by any chance,  may be it is better to sleep only if isTableEnabled() call comes back as disabled.  The concern is create table will now sleep for one second always.